apps/teams-test-app/e2e-test-data/externalAppAuthenticationForCEA.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ExternalAppAuthenticationForCEA",
+  "version": ">2.29.0",
+  "platform": "Web",
+  "testCases": [
```

Comment:
>testCases [](http://example.com/codeflow?start=3&length=9)

Does there have to be a web host SDK version restriction for this, since the support for this capability was only recently added?

Reply:
I don't think I understand. The platform "Web" will restrict it to web. What else do I need?

Reply:
Since we support multiple versions of the host sdk simultaneously, the automated tests are run against those versions when new releases are needed. Because the implementation of this capability does not exist in all of those versions, I think you will need to restrict which host SDK versions this test runs against using a filter like this:
```
  "hostSdkVersion": {
    "web": ">2.16.0"
  },
```

If you search for "hostSdkVersion" you can find some examples in the other JSON files.

Is that clearer?

Reply:
that makes sense

Reply:
If this was checking against any lower version than when teh support was added (4.3.0), should not it have failed in teh pipelines. I am adding the version to be safe :)

Reply:
Yes, it "should" fail in the pipelines but as of right now the tests are currently only run on the older releases "on demand." This is obviously not supportable because it will lead to people such as yourself making perfectly innocent changes.

There is a bug tracking enabling these older tests in the pipelines that is sitting on Jay's plate but for privacy reasons I cannot provide the bug link to you here.